go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/honeycombio/dynsampler-go
```

Comment:
I believe this is the correct module name given historical distribution via this GitHub repo. Yes?

Reply:
Should be. [Refinery go.mod has `github.com/honeycombio/dynsampler-go v0.2.1`](https://github.com/honeycombio/refinery/blob/main/go.mod#L17)

Reply:
🧐  Indeed. And @lizthegrey said "yes" out-of-band and she Knows Things™.